### PR TITLE
feat: Implemented image upload on summernote and cover image for article

### PR DESCRIPTION
**Pull request by yspanchal**

### CODE REVIEW

File: newscout_web/api/v1/urls.py [hunk 1]
```diff
@@ -67,6 +68,8 @@
         name='article-create-update'),
     url(r'^article/status/$', ChangeArticleStatusView.as_view(),
         name='article-status'),
-    url(r'daily-digest/$', GetDailyDigestView.as_view(), name='daily-digest')
+    url(r'daily-digest/$', GetDailyDigestView.as_view(), name='daily-digest'),
+    url(r'article/draft-image/$', DraftMediaUploadView.as_view(), name='draft-image'),
+    url(r'article/draft-image/(?P<pk>[-\d]+)/$', DraftMediaUploadView.as_view(), name='draft-image'),
```
Comment (sidharthshah) [inline newscout_web/api/v1/urls.py]: Consolidate `article/draft-image` and `article/draft-image/(?P<pk>[-\d]+)` into one

File: newscout_web/api/v1/views.py [hunk 2]
```diff
@@ -1020,6 +1020,15 @@ def publish(self, obj):
 
     def post(self, request):
         publish = request.data.get("publish")
+        origin = request.META.get("HTTP_ORIGIN")
```
Comment (sidharthshah) [inline newscout_web/api/v1/views.py]: Just comment the usage of `origin` in the code

File: newscout_web/api/v1/views.py [hunk 3]
```diff
@@ -1160,3 +1177,41 @@ def list(self, request, *args, **kwargs):
             return Response(create_response(serializer.data))
         else:
             return Response(create_error_response({"Msg" : "Daily Digest Doesn't Exist"}), status=400)
+
+
+class DraftMediaUploadView(APIView):
```
Comment (sidharthshah) [inline newscout_web/api/v1/views.py]: Please also include API tests for this view @rsqwerty

File: newscout_web/api/v1/views.py (hunk 2 above)
Comment (yspanchal) [reply newscout_web/api/v1/views.py]: fixed in commit id: b336df54a91d0a1f30b64f8b703039aff6191f5d

**Pull request merged**